styles/components/_select2.scss

```diff
@@ -84,10 +93,13 @@
     // This class must be set via JS
     &.emptyOfTags {
       display: block;
-    }    
+      top: 0;
+      line-height: 4rem;
```

Comment:
The line height is set to 4rem here, because that's what the original select2 css sets elements in the control to.